blocksync/buffer.go

```diff
@@ -35,7 +35,7 @@ func newBlockBuffer(bufferSize, intervalSize uint64) *blockBuffer {
 	}
 }
 
-func (b *blockBuffer) Delete(height uint64) []*peerBlock {
+func (b *blockBuffer) Pop(height uint64) []*peerBlock {
```

Comment:
why do you rename this function?

Reply:
`Delete` is to remove buffer of height x, whereas `Pop` is to remove buffer of height x and return the buffer to the caller.